### PR TITLE
19451: Add ability to upload SCO file to GIDS

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As an EDU team member, I need the ability to upload School Certifying Official data to GIDS so that it can be used to display information in the Comparison Tool.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19451

## Testing done
Local manual and unit/spec tests pass successfully, additional unit test added for new file type.

## Acceptance criteria
- [x] A user can upload a .csv file containing SCO data matching the format outlined in Supporting Artifact # 1 into the GIDS Tool.
- [x] A user can export the most recently uploaded SCO data from the GIDS Tool.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
WeamsCertifyingOfficials should be singular.

File: app/models/weams_certifying_officials.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class WeamsCertifyingOfficials < ActiveRecord::Base
+  include CsvHelper
+
+  COLS_USED_IN_INSTITUTION = %i[closure109].freeze
```
Comment (dmccurdybah) [inline app/models/weams_certifying_officials.rb]: I think this can be removed

File: spec/models/version_spec.rb [hunk 2]
```diff
@@ -10,8 +10,9 @@
     let(:after_preview_date) { Faker::Time.between(preview_date + 1.day, preview_date + 2.days).utc }
     let(:before_preview_date) { Faker::Time.between(preview_date - 2.days, preview_date - 1.day).utc }
     # TODO: this is brittle, Version.buildable_state should do a proper check:
-    let(:upload_dates_after) { [after_preview_date] * 30 }
-    let(:upload_dates_before) { [before_preview_date] * 30 }
+
```
Comment (shawkey-daniel) [inline spec/models/version_spec.rb]: Please remove this extra whitespace

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: spec/models/version_spec.rb (hunk 2 above)
Comment (ksantiagomorgan6) [reply spec/models/version_spec.rb]: White space removed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by jholton — APPROVED** [review]
LGTM

**Comment by ksantiagoBAH** [comment]
updated headers

https://github.com/department-of-veterans-affairs/gibct-data-service/pull/434

**Pull request merged**